Library/ViewModels/MessageBannerViewModel.swift

```diff
@@ -3,11 +3,12 @@ import Prelude
 import ReactiveSwift
 import Result
 
+public typealias MessageBannerConfiguration = (type: MessageBannerType, message: String)
+
 public protocol MessageBannerViewModelInputs {
-  func setBannerType(type: MessageBannerType)
-  func setBannerMessage(message: String)
-  func showBannerView(shouldShow: Bool)
   func bannerViewAnimationFinished(isHidden: Bool)
+  func showBannerView(shouldShow: Bool)
```

Comment:
Can I suggest another name for this input to be `bannerViewVisibilityWillChange(toShow show: Bool)` just to keep with conventions of outputs generally being _what_ the effect of the input is.